bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -106,7 +107,10 @@ final class BagsManager(executorService: ExecutorService) {
         case AutomaticGroup.Initial | AutomaticGroup.Active(_,_) => true
         case _                                                   => false
       }
-      enabledGroup && (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI)
+      enabledGroup &&
+        (ctx.getInstrument.getType != SPComponentType.INSTRUMENT_GPI) &&
```

Comment:
GPI doesn't need a guide star, that may be equivalent to make it use `ScienceTargetStrategy`, or we could add an entry to `Strategy.InstMap` for GPI returning an empty list